src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -1029,6 +1030,10 @@ bool fuse_type_to_constant(const std::shared_ptr<ngraph::Node>& node,
                            const precisions_map& precisions,
                            const std::vector<Input<Node>>& consumers) {
     auto from = node->get_element_type();
+    // Consts marked with disable_constant_folding should be kept in f16 until they reach to the CPU plugin
```

Comment:
I believe that will be required not only for CPU plugin, so I would propose to remove any plugin specification here.

Reply:
done